shared/checks/oval/installed_env_has_ovirt-host_package.xml

```diff
@@ -0,0 +1,38 @@
+<def-group>
+
+  <definition class="inventory"
+  id="installed_env_has_ovirt-host_package" version="1">
+    <metadata>
+      <title>Package ovirt-host is installed</title>
+      <affected family="unix">
+        <platform>multi_platform_all</platform>
+      </affected>
+      <description>Checks if package ovirt-host is installed.</description>
+      <reference ref_id="cpe:/a:ovirt-host" source="CPE" />
+    </metadata>
+    <criteria>
+      <criterion comment="Package ovirt-host is installed" test_ref="test_env_has_ovirt-host_installed" />
+    </criteria>
+  </definition>
+
+{{% if pkg_system == "rpm" %}}
+  <linux:rpminfo_test check="all" check_existence="at_least_one_exists"
+  id="test_env_has_ovirt-host_installed" version="1"
+  comment="system has package ovirt-host installed">
+    <linux:object object_ref="obj_env_has_ovirt-host_installed" />
+  </linux:rpminfo_test>
+  <linux:rpminfo_object id="obj_env_has_ovirt-host_installed" version="1">
+    <linux:name>ovirt-host</linux:name>
+  </linux:rpminfo_object>
+{{% elif pkg_system == "dpkg" %}}
```

Comment:
We don't package for dpkg, this check can be removed if it's not required by the format.

Reply:
@almusil So there are no Ubuntu or Debian packages for oVirt?

Reply:
That's correct, we are officially building only for rpm targets.

Reply:
Removed the dpkg tests.